repo/config/init.go

```diff
@@ -105,20 +115,31 @@ func Init() (*Config, error) {
 			Profiling: "127.0.0.1:6060",
 		},
 		API: API{
-			HTTPHeaders: map[string][]string{
+			HTTPHeaders: HTTPHeaders{
 				"Server": {"go-textile/" + common.Version},
+				// Explicitly allow all methods
 				"Access-Control-Allow-Methods": {
-					"GET",
-					"POST",
-					"DELETE",
-					"OPTIONS",
+					http.MethodConnect,
+					http.MethodDelete,
+					http.MethodGet,
+					http.MethodHead,
+					http.MethodOptions,
+					http.MethodPatch,
+					http.MethodPost,
+					http.MethodPut,
+					http.MethodTrace,
 				},
 				"Access-Control-Allow-Headers": {
+					// rs/cors default headers
+					"Origin",
+					"Accept",
 					"Content-Type",
+					"X-Requested-With",
+					// reason why this is here is unknown
 					"Method",
```

Comment:
Ha, past copy-pasta error?

Reply:
nope, it is there, is a not for me having no idea why `Method` is there